sros2/sros2/verb/create_permission.py

```diff
@@ -31,17 +31,23 @@ class CreatePermissionVerb(VerbExtension):
     """Create permission."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('ROOT', help='root path of keystore')
+        arg = parser.add_argument(
+            '-k', '--keystore-root-path',
+            help='root path of keystore')
         arg.completer = DirectoriesCompleter()
-        parser.add_argument('NAME', help='key name, aka ROS node name')
+        parser.add_argument(
+            '-c', '--security-contexts', nargs='*', default=[],
```

Comment:
Same as above this is changing behavior and semantic

Reply:
Addressed in https://github.com/ros2/sros2/pull/177/commits/103e18c1366ed793c6269f2d9adab8a16aaf7046.